api/tests/certification/enrolment/integration/infrastructure/repositories/certification-candidate-repository_test.js

```diff
@@ -41,6 +42,16 @@ describe('Integration | Repository | CertificationCandidate', function () {
         expect(_.omit(firstCertificationCandidatesInSession, attributesToOmit)).to.deepEqualInstance(
           _.omit(certificationCandidate, attributesToOmit),
         );
+        const subscriptions = await knex('certification-subscriptions').where({
+          certificationCandidateId: firstCertificationCandidatesInSession.id,
+        });
+
+        expect(subscriptions).to.have.lengthOf(1);
+        expect(subscriptions[0]).to.contain({
+          type: SubscriptionTypes.CORE,
+          certificationCandidateId: firstCertificationCandidatesInSession.id,
+          complementaryCertificationId: null,
+        });
```

Comment:
Le .contain n'est pas sur un tableau, pourquoi l'utiliser comme ca?
```suggestion
        expect(subscriptions).to.have.lengthOf(1);
        expect(subscriptions).to.have.deep.members({
          type: SubscriptionTypes.CORE,
          certificationCandidateId: firstCertificationCandidatesInSession.id,
          complementaryCertificationId: null,
        });
```

Reply:
Done !